### PR TITLE
Fix pipeline docker logger level

### PR DESCRIPTION
**Pull request by fermadeiral**
I have not obtained debug log messages from the pipeline when running it from dockerpool. This PR proposes a solution for that. I changed only for Bears because I don't know if you are interested on this. If you are, and if you like the proposed changes, let me know, then I'll do the changes for Repair too.

### CODE REVIEW

File: repairnator/docker-images/bears-dockerimage/logback.xml [hunk 1]
```diff
@@ -14,15 +14,15 @@
 	</appender>
 
 	<logger name="fr.inria.spirals.jtravis.helpers" level="info" />
-	<logger name="fr.inria.spirals.repairnator" level="info" />
+	<logger name="fr.inria.spirals.repairnator" level="${LOG_LEVEL:-info}" />
```
Comment (surli) [inline repairnator/docker-images/bears-dockerimage/logback.xml]: I don't understand this notation. The ":-info" is a default value?

File: repairnator/docker-images/bears-dockerimage/logback.xml [hunk 2]
```diff
@@ -14,15 +14,15 @@
 	</appender>
 
 	<logger name="fr.inria.spirals.jtravis.helpers" level="info" />
```
Comment (surli) [inline repairnator/docker-images/bears-dockerimage/logback.xml]: It should be changed to `fr.inria.jtravis`

File: repairnator/docker-images/bears-dockerimage/logback.xml (hunk 1 above)
Comment (fermadeiral) [reply repairnator/docker-images/bears-dockerimage/logback.xml]: Yes.

File: repairnator/docker-images/bears-dockerimage/logback.xml (hunk 2 above)
Comment (fermadeiral) [reply repairnator/docker-images/bears-dockerimage/logback.xml]: Done.

**Pull request merged**